rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -68,6 +69,24 @@ static FORCE_INLINE void mov_rdata(decltype(spu_thread::rdata)& dst, const declt
 	}
 }
 
+// Returns nullptr if rsx does not need pausing on reservations op, rsx ptr otherwise
+static FORCE_INLINE rsx::thread* is_need_rsx_reservations_pause(u32 addr)
```

Comment:
The naming of this method can be improved a bit. The current name makes it feel like it returns a bool, but it returns a thread object. Name it get_xxx something to pass on this idea. For example, I had to check again to be sure I was reading right when checking the use cases below.
```
const auto render = is_need_rsx_reservations_pause(addr);
if (render) render->pause();
```
That is confusing to read.

Reply:
If you want a short name, try something like `get_rsx_reservations` or `apply_rsx_reservations` or something along those lines. Just make it not look like its bool.

Reply:
Name is still long but less confusing.